backend/tests/Feature/TablePageViewsApiTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Entities\GeoPosition;
+use App\Entities\Page;
+use App\Entities\Session;
+use App\Entities\System;
+use App\Entities\User;
+use App\Entities\Visit;
+use App\Entities\Visitor;
+use App\Entities\Website;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class TablePageViewsApiTest extends TestCase
+{
+    use RefreshDatabase;
+
+    private const RESPONSE_STRUCTURE = [
+        'data' => [
+            '*' => [
+                'page_url',
+                'page_title',
+                'count_page_views',
+                'bounce_rate',
+                'exit_rate'
+            ]
+        ],
+        'meta' => []
+    ];
+
+    private const URL = 'api/v1/table-page-views';
+
+    private $user;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->user = factory(User::class)->create();
+        factory(Website::class)->create();
```

Comment:
if you don't set values for test explicit your test is probably to fail. Because you do not know what will change in factroies, and some parameters there are set randomly.

Reply:
now looks better